libbeat/_meta/config.yml

```diff
@@ -91,3 +91,18 @@ output.elasticsearch:
 # To enable all selectors use ["*"]. Examples of other selectors are "beat",
 # "publish", "service".
 #logging.selectors: ["*"]
+
+#============================== Xpack Monitoring ===============================
+# beatname can export internal metrics to a central Elasticsearch monitoring
+# cluster.  This requires xpack monitoring to be enabled in Elasticsearch.  The
+# reporting is disabled by default.
+
+# Set to true to enable the monitoring reporter.
+#xpack.monitoring.enabled: false
+
+# Uncomment to send the metrics to Elasticsearch. Most settings from the
+# elasticsearch output are accepted here as well. Any setting that is not set is
+# automatically inherited from the output configuration, so if you want to use
+# the same cluster for monitoring data, you can simply uncomment the following
+# line.
```

Comment:
x-pack always requires to pass data through production cluster. That is hosts must match the elasticsearch output.

>  so if you want to use the same cluster for monitoring data, you can simply uncomment the following
 line.

Note: elasticsearch report configs are only inherrited from output.elasticsearch config. If logstash is configured as output, the host+username+password for ES must be configured here.

Reply:
Ah, I forgot about that. Let's see if I can clarify it.